test/parsing.test.ts

```diff
@@ -1149,6 +1178,36 @@ describe('A Markdown parser function', () => {
                 ],
             },
         },
+        'link with title and escaped quote': {
```

Comment:
Should we add a test for escaped parenthesis too?

Reply:
It's already covered [here](https://github.com/croct-tech/md-lite-js/blob/master/test/parsing.test.ts#L1031-L1088).